mining_test.go

```diff
@@ -45,7 +45,7 @@ func TestTxFeePrioHeap(t *testing.T) {
 		txpi, ok := prioItem.(*txPrioItem)
 		if ok {
 			if last.feePerKB < txpi.feePerKB &&
```

Comment:
Also notice the error message say "fee per KB was **more than**".  Making the condition match the error not only fixes the logic error, but reads better too.